src/client/index.html

```diff
@@ -2,6 +2,9 @@
 <html lang="en">
 
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-137652528-1"></script>
+    <script src="./gtagSupport.js"></script>
```

Comment:
So as mentioned above, I think this code will work better if it is placed into modules/app/services/analytics.js. That service has a hook to catch navigation events, which is a good place to trigger page hits, because it is only triggered after auth is set up, so the username is available. It also provides the navigation path, so you don't need to extract it from the url. Most of the contents of analytics.js, the parts which directly invoke the measurement api and place the GA compatible cookies, can be removed, being replaced with the tag calls.
Another benefit is that the first page hit can be handled and will have the username (onpopstate won't be invoked on the first page hit.)